src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -89,12 +89,24 @@ public function getTranslationNodeVisitor(): TranslationNodeVisitor
         return $this->translationNodeVisitor ?: $this->translationNodeVisitor = new TranslationNodeVisitor();
     }
 
-    public function trans(string $message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
+    public function trans($message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
     {
         if (null !== $count) {
             $arguments['%count%'] = $count;
         }
 
-        return $this->getTranslator()->trans($message, $arguments, $domain, $locale);
+        if ('' === $message || [] === $messages = (array) $message) {
+            return '';
+        }
+
+        $translator = $this->getTranslator();
+
+        foreach ($messages as $msg) {
```

Comment:
1. This does not mean the message is not translated
2. I'm not sure the $arguments, $domain, $locale will be the same in every situation

Reply:
1. right now, passing an array is not allowed. So yes, it means the message is not translated, because we just defined it that way
2. then you'd need another interface, which this PR doesn't aim to solve.

Reply:
1 .Oups, I commented on the wrong line :/ It should have been on the next line.
  ```
  `$msg !== $message = $translator->trans()`
  ```
  > because we just defined it that way
  ATM, this behavior is not documented
2. OK

Reply:
Explanation added in the docblock.